src/elements/emby-scroller/emby-scroller.scss

```diff
@@ -26,3 +26,7 @@
         margin-left: 1.2em;
     }
 }
+
+.no-padding-left {
+    padding-left: 0;
```

Comment:
We need `padding-right: 0;` for Right-To-Left text direction.

Reply:
I believe I have updated this now. It is more generic and looks for the direction and sets padding to be 0 on the correct side accordingly